src/packages/@teams/features/team-members/components/member/Member.svelte

```diff
@@ -296,10 +303,20 @@
     class="d-flex align-items-center justify-content-between gap-3 mt-1 pb-3 mb-0 rounded"
     style="font-size: 16px;"
   >
-    <div class="d-flex align-items-center ellipsis">
+    <div class="d-flex align-items-center ellipsis gap-2">
       {#if openTeam?.logo}
-        <img class="team-icon me-2" src={base64ToURL(openTeam?.logo)} alt="" />
-      {/if}
+      <img class="team-icon me-2" src={base64ToURL(openTeam?.logo)} alt="" />
+    {:else}
+      <div
+        class="d-flex align-items-center justify-content-center"
+        style="width: 36px;
+      border: 1px solid var(--border-color);
+      height: 36px;
+      border-radius: 50%;"
+      >
+        <span>{openTeam?.name[0].toUpperCase()}</span>
+      </div>
+    {/if}
```

Comment:
lets convert it to reusable component named `IconFallback` in "library/ui" using props - 
height, width, backgroundColor, borderColor, textColor, fontSize

use js docs, and hocs